torch/_inductor/codegen/cpp.py

```diff
@@ -1783,6 +1880,11 @@ def __init__(
         self.tiling_idx = tiling_idx
         metrics.generated_cpp_vec_kernel_count += 1
 
+    def _reduction_var_name(self, name_prefix: str, line: str) -> str:
+        if f"{name_prefix}_vec" in line:
```

Comment:
what happens if we don't check this?

Reply:
Perhaps you can revise the regular expression to catch both: `re.search(r'\btmp_acc_?[a-z]*\d+\b', line)`

Reply:
`re.search("tmp_acc[0-9]+?_vec", line)` is able to work for both but I am trying to not put `vec` thing into `CPPKernel`.
I revised it by using `re.search("tmp_acc[0-9]+", line)` and `re.search("tmp_acc[0-9]+_vec", line)` separately.